manifests/init.pp

```diff
@@ -203,6 +206,7 @@
   Stdlib::Host $artemis_host = $candlepin::params::host,
   Stdlib::Port $artemis_port = $candlepin::params::artemis_port,
   String $artemis_client_dn = $candlepin::params::artemis_client_dn,
+  Boolean $manage_certificates = $candlepin::params::manage_certificates,
```

Comment:
Slightly more explicit would be:
```suggestion
  Boolean $manage_certificate_files = $candlepin::params::manage_certificate_files,
```

This doesn't give the impression that we also manage the content of those files.